src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer" + " " + machineType + " " + "started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer" + " " + machineType + " " + "stopped working.");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative. Rather than just saying 'stopped working', it could be more descriptive, like 'Bulldozer has finished moving earth'.